content/zotero_intro_to.md

```diff
@@ -242,13 +242,13 @@ Open Zotero and click on the "Save to Zotero" button in your browser to save the
 
 Example : [an article from Journal of Wine Research](https://doi.org/10.1080/09571264.2018.1433140)
 
-**|!| Tips : choose the right source of information and do not procrastinate**
+**💡 Tips : choose the right source of information and do not procrastinate**
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok